### PR TITLE
Update manage.py

### PR DESCRIPTION
**Pull request by Guru-25**

### CODE REVIEW

**Comment by Guru-25** [comment]
@Guru-25 Yup

**Comment by Guru-25** [comment]
@Guru-25

**Comment by Guru-25** [comment]
@Guru-25 😒😐😒😐😒

File: userge/plugins/tools/manage.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ async def status(message: Message) -> None:
     name_ = message.filtered_input_str
     type_ = list(message.flags)
     if not type_:
-        out_str = f"""📊 **--Userge Status--** 📊
+        out_str = f"""📊 **--Userge-X Status--** 📊
```
Comment (code-rgb) [inline userge/plugins/tools/manage.py:39]: USERGE-X

Comment (Guru-25) [reply userge/plugins/tools/manage.py:39]: Ok

**Pull request closed without merging**